core/src/main/java/hudson/PluginManager.java

```diff
@@ -1703,7 +1703,7 @@ public HttpResponse doUploadPlugin(StaplerRequest req) throws IOException, Servl
 
     @Restricted(NoExternalUse.class)
     @RequirePOST public HttpResponse doCheckUpdatesServer() throws IOException {
-        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+        Jenkins.get().checkPermission(Jenkins.SYSTEM_READ);
```

Comment:
Not sure about that. It is not a read-only operation

Reply:
This was called out in the previous PR and dB was fine with it, it’s updating a cache, not updating configuration or installing something